### DISCUSSION TITLE
New UI shows limited info for video download selection

### DISCUSSION THREAD

**Discussion by cnwtech** [aclap-dev/video-downloadhelper#752] [Help and Q&A]
### Your question

OLD UI showed various videos available for download on specific web page.
In NEW UI, I dont see all of the available videos (different formats & size) nor duration (to distinguish different videos on page).   This makes it very difficult to select correct format/size desired, and to ensure specific download option corresponds to specific video of many on web page.   

For example, 2 videos A & B may be on the same web page, available in different download formats (640x480, 1024x768, mkv vs mp4 etc) and in the past, I could differentiate one from other based on DURATION HH:MM and name the downloaded video accordingly.  

NOW... it gives 2 videos in VDH options to download... each with the same filename... but no info on duration to determine which is which.   This means I have to wait until they download, then manually determine by playing video and then renaming file.... OMG... too much work!

How can I work around the new UI?

I see SETTINGS has options to show lower resolution formats.... so I turned this on... but I still can't see more info about DURATION of file... as I used t o be able to see.

If you want to see regression... this issue occurs on YouTube videos... on FireFox (latest), as well as wistia and (all) others....
EXAMPLE:   https://www.youtube.com/watch?v=Zm255g3URpw 
(see photo for plug-in info... no duration like in old UI)

![Screenshot 2024-05-28 at 4 33 30 PM](https://github.com/aclap-dev/video-downloadhelper/assets/36424620/d94b3a48-176c-4dc2-9a18-0680ec70a646)

This kinda breaks usability for me as it makes process very MANUAL.

Please advise... thanks! G

### Addon & CoApp version and details

version: 9.0.1.13
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"11909f684a654deb"}
platform: aarch64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "/Users/grantl/Downloads/VDH",
    "view_options": {
        "all_tabs": false,
        "low_quality": true,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 34
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0.1 64-bit, licensed VDH 9.0.2.3a1 beta, CoApp 2.0.19.

I can't address your issue with pages that have multiple videos on them.  I did visit your YouTube example & I got this:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/62e7ab07-823d-45d4-938f-de78cf9642a1)

Looks fine to me.  I didn't actually try to download it.  First, it's long.  Second, I'm not interested in the content.

I have observed on occasion that VDH does not do a great job of recording YouTube livestreams.  But if you wait until the livestream ends & they put up an archival version of it, which appears to be the case with this YouTube example, then VDH can download it no problem.  I have also noticed that if you look at something, anything, on YouTube too soon after they post it, not all the resolutions are available.  If you wait just a few minutes, the rest of the resolutions appear.

There is also this in the VDH settings after you click the More settings option:

![image](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/fe9885bd-6685-468a-8b4d-59a99b963c3a)

Try bumping up that Variants count setting.  You have to scroll down that secondary page of settings to find that.

For your multi-video pages, I would take a look at one if you give the URL of an example.   As long as it doesn't require a login, I'll look at it.

**Comment by jamesritzman** [comment]
I was having the same issue, but then I found that in More Settings, down at the bottom (you may need to scroll), there is an option to use the "Legacy UI," which I prefer myself.

**Comment by paulrouget** [comment]
@cnwtech I'm confused… your screenshot shows the duration. What's wrong with that?

<img width="289" alt="Screenshot 2024-05-30 at 01 31 01" src="https://github.com/aclap-dev/video-downloadhelper/assets/373579/bcc88a3c-67ee-4e65-9816-51379dd91ef3">
